### PR TITLE
[now-cli] Add 1 minute timeout to `now login` integration tests

### PR DESCRIPTION
**Pull request by TooTallNate**

### CODE REVIEW

File: packages/now-cli/test/integration.js [hunk 1]
```diff
@@ -209,6 +209,8 @@ test.after.always(async () => {
 });
 
 test('login', async t => {
+  t.timeout(ms('10s'));
```
Comment (lucleray) [inline packages/now-cli/test/integration.js]: Isn't 10 seconds a bit low for login? I'd put 30 seconds.

Comment (styfle) [reply packages/now-cli/test/integration.js]: Same 10 sec seems too low for email

**Review by lucleray — APPROVED** [review]
Thanks! This will make running tests much better 😍

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**